man/portage.5

```diff
@@ -667,6 +667,9 @@ overriding the value from \fBmake.conf\fR(5).
 Specifies priority of given repository. When a package exists in multiple
 repositories, those with higher priority are preferred.
 .TP
+.B location
+Specifies location of repository file cache.
```

Comment:
We should figure out a suggested location in future (and maybe a default later on than that) but not something to worry about for now.

Reply:
I realized that we could handle it transparently if the user happened to specify the same location as PKGDIR here, so I updated the docs like this:
```diff
+.B location
+Specify a cache location which serves as a partial local mirror of a
+remote repository. If unset then the effective default is PKGDIR
+(see \fBmake.conf\fR(5)).
+.TP
```